packages/docs/src/routes/docs/cheat/anti-patterns/index.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Anti-patterns
+contributors:
+  - mhevery
+---
+
+# No-Hydration Tips
+## Don't register events eagerly with `useClientEffect$()`
```

Comment:
How about changing this to `Don't register DOM events eagerly with `useClientEffect$()`? I was confused until I realized it was about DOM

Reply:
@KenAKAFrosty

Reply:
@mhevery